pkg/storkctl/clusterdomainupdate.go

```diff
@@ -195,3 +227,32 @@ func clusterDomainUpdatePrinter(cduList *storkv1.ClusterDomainUpdateList, writer
 	}
 	return nil
 }
+
+func waitForDomainUpdate(name string) string {
+	var msg string
+
+	log.SetFlags(0)
+	log.SetOutput(ioutil.Discard)
```

Comment:
This needs to be set to ioStreams.Out I think

Reply:
Actually, why do we need this?

Reply:
to suppress logs in `DoRetryWithTimeout`